docs/articles/documentation/using-testcafe/command-line-interface.md

```diff
@@ -0,0 +1,215 @@
+---
+layout: docs
+title: Command Line Interface
+permalink: /documentation/using-testcafe/command-line-interface/
+---
+# Command Line Interface
+
+```sh
+testcafe [options] <browser-list-comma-separated> <file-or-glob ...>
+```
+
+## Browser List
+
+### Local Browsers
+
+Specify the list of browsers (separated by commas) where tests will be ran.
+You can specify browser paths or [aliases](/testcafe/documentation/using-testcafe/common-concepts/browser-aliases). 
+A list of all the available aliases can be obtained by the [--list-browsers](#b) command.
+
+The following example demonstrates how to run a test in several browsers. The browsers can be specified by aliases as well as by paths.
```

Comment:
> The browsers **can be** specified by aliases as well as by paths.

If you are talking about the sample, 'the browsers **are** specified'
If you are saying this in general, this should belong to the previous paragraph (which already says this, BTW)